src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ContigAligner.java

```diff
@@ -54,52 +51,106 @@ public ContigAligner(final String referenceFilePath) throws IOException {
         log.info("Created BWA MEM");
     }
 
-    public List<AssembledBreakpoint> alignContigs (final ContigsCollection contigsCollection) {
-        final List<AssembledBreakpoint> assembledBreakpoints = new ArrayList<>();
+    public List<AlignmentRegion> alignContigs(String breakpointId, final ContigsCollection contigsCollection) {
```

Comment:
I believe this method, as the core method, needs a documentation for 
1. what it does (kind of obvious)
2. what type filters are applied to the alignment regions (no secondary?)
3. any specific order in the returned list (start position in contig?)

And the `String` parameter could be final.